src/server/Server.php

```diff
@@ -99,6 +102,9 @@ public function __construct(
 		$this->socket->setBlocking(false);
 
 		$this->unconnectedMessageHandler = new UnconnectedMessageHandler($this, $protocolAcceptor);
+
+		// If you don't want to use security on the server, just delete this line.
+		$this->cookie = new Cookie();
```

Comment:
Users of the library don't want to change the library itself to change a behavior, a switch in the constructor need to be created